learning_journal/learning_journal/templates/edit.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "index.jinja2" %}
+{% block body %}
+  <form action="{{ request.route_url('update', id=entry.id) }}" method="POST">
+    <div class="form-group col-xs-10">
+      <h2>Title</h2>
+      <input type="text" class="form-control" name="title" value="{{ entry.title }}"><br>
```

Comment:
In general, I find that using `<br>` in your html means you haven't used your CSS to solve a problem that should have been solved there.